src/rpc/util.cpp

```diff
@@ -65,11 +65,8 @@ void RPCTypeCheckObj(const UniValue& o,
         if (!fAllowNull && v.isNull())
             throw JSONRPCError(RPC_TYPE_ERROR, strprintf("Missing %s", t.first));
 
-        if (!(t.second.typeAny || v.type() == t.second.type || (fAllowNull && v.isNull()))) {
```

Comment:
Any reason to remove the `{}`? According to the dev notes we want them

Reply:
> Any reason to remove the {}? According to the dev notes we want them

Yes, this would also reduce the diff.